cedar-policy-validator/src/schema.rs

```diff
@@ -522,34 +519,22 @@ impl ValidatorSchema {
             .flat_map(move |e| self.get_entities_in(var, e))
     }
 
-    /// Since different Actions have different schemas for `Context`, you must
-    /// specify the `Action` in order to get a `ContextSchema`.
+    /// Get the `Type` of context expected for the given `action`.
+    /// This always reutrns a closed record type.
     ///
     /// Returns `None` if the action is not in the schema.
-    pub fn get_context_schema(
-        &self,
-        action: &EntityUID,
-    ) -> Option<impl cedar_policy_core::entities::ContextSchema> {
-        self.get_action_id(action).map(|action_id| {
-            // The invariant on `ContextSchema` requires that the inner type is
-            // representable as a schema type. Here we build a closed record
-            // type, which are representable as long as their values are
-            // representable. The values are representable because they are
-            // taken from the context of a `ValidatorActionId` which was
-            // constructed directly from a schema.
-            ContextSchema(crate::types::Type::record_with_attributes(
-                action_id
-                    .context
-                    .iter()
-                    .map(|(k, v)| (k.clone(), v.clone())),
-                OpenTag::ClosedAttributes,
-            ))
-        })
+    pub fn context_type(&self, action: &EntityUID) -> Option<Type> {
+        // INVARIANT: `ValidatorActionId::context_type` always returns a closed
```

Comment:
nit: I don't think we need to repeat this since it's in the docstring above.

Reply:
The docstring above is to document `ValidatorSchema::context_type`'s return invariant.  The comment here is explaining how we meet that invariant, namely by discharging it to `ValidatorActionId::context_type`.